cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -888,6 +888,31 @@ Slice::CsGenerator::writeUnmarshalCode(Output &out,
     }
 }
 
+namespace
+{
+bool isVariableLength(TypePtr type)
+{
+    BuiltinPtr builtin = BuiltinPtr::dynamicCast(type);
+    ProxyPtr proxy = ProxyPtr::dynamicCast(type);
```

Comment:
Also doesn't look like `proxy` gets used anywhere.

Reply:
Also no longer applicable since you switched to the `Type::isVariableLength` method.

Reply:
I removed the method and go with Type::isVariableLength